src/classes/CW_AudienceCSVUploadController.cls

```diff
@@ -0,0 +1,313 @@
+/**
+* File Name   :    CW_AudienceCSVUploadController.cls
+* Description :    B-109507
+*                  upload CSv with groups/group memebers
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 02/04/2014 Klea Kolaric   Initial version
+* 0.2 29.04.2014 AleX Panka     Add shells Person Account support
+* 0.3 25.05.2014 Alex Panka     Audience groups rewards 
+* 0.4 08.08.2014 Cristina P		Upload Audience Group and Members in the same time
+*
+*/
+//Cristina P: 25/06/14: Changing the class from "with sharing" to "without sharing" 
+//so the user can create related records while cost/brief is in approval
+public without sharing class CW_AudienceCSVUploadController 
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.